api/v3/Birthdays.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/*
+ * Copyright (C) 2022 SYSTOPIA GmbH
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation in version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+/**
+ * birthdays.sendgreetings
+ *
+ * @param array $params
+ *   API call parameters
+ *
+ * @return array
+ *   API3 response
+ *
+ */
+function civicrm_api3_birthdays_sendgreetings(array $params): array
+{
+    try {
+        $results = \Civi\Api4\Birthdays::sendGreetings()
+            ->execute();
+        $return_info = [];
+
+        foreach ($results as $results_sub) {
```

Comment:
Why has this two levels of nesting and why do we need to re-format them like this? Maybe let's add an example in a comment to understand what this actually returns.